workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga

```diff
@@ -0,0 +1,1165 @@
+            ],
+            "size": [
+                461.7,
+                234.8
+            ],
+            "type": "frame"
+        },
+        {
+            "child_steps": [
+                2,
+                4,
+                6,
+                7,
+                9,
+                12,
+                16
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Extracting Contigs from our Samples"
+            },
+            "id": 1,
+            "position": [
+                310,
+                490.20000000000005
+            ],
+            "size": [
+                1468.9,
+                676
+            ],
+            "type": "frame"
+        },
+        {
+            "child_steps": [
+                1,
+                3,
+                5,
+                8
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Extract Samples IDs"
+            },
+            "id": 0,
+            "position": [
+                302.4,
+                0.0
+            ],
+            "size": [
+                1060.5,
+                234.8
+            ],
+            "type": "frame"
+        }
+    ],
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "0000-0001-9047-4215",
+            "name": "Engy Nasr",
+            "url": "https://orcid.org/0000-0001-9047-4215"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0001-9852-1987",
+            "name": "B\u00e9r\u00e9nice Batut",
+            "url": "https://orcid.org/0000-0001-9852-1987"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0003-2982-388X",
+            "name": "Paul Zierep",
+            "url": "https://orcid.org/0000-0003-2982-388X"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "release": "0.1",
+    "name": "Gene-based-Pathogen-Identification",
+    "report": {
+        "markdown": "# Nanopore Pathogen Detection - Gene based pathogenic identification Workflow Report\nBelow are the results for the Gene based pathogenic identification Workflow\n\nThis workflow was run on:\n\n```galaxy\ngenerate_time()\n```\n\nWith Galaxy version:\n\n```galaxy\ngenerate_galaxy_version()\n```\n\n## Workflow Inputs\nThe per-processing workflow main output (All samples reads after quality retaining and hosts filtering), and the MLST header report\n\n## Workflow Output\n\r\n```galaxy\ninvocation_outputs()\n```\r\n"
+    },
+    "steps": {
+        "0": {
+            "annotation": "Nanopore Sequenced Reads Collection",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Nanopore Sequenced Reads Collection",
+                    "name": "nanopore_sequenced_reads_collection"
```

Comment:
Could you use explicit names and description here and in every workflows?
They are used after to populate the "Inputs" table on WorkflowHub page, e.g. https://workflowhub.eu/workflows/870?version=1